app/build.gradle

```diff
@@ -18,6 +18,13 @@ android {
     dexOptions {
         javaMaxHeapSize "4g"
     }
+    compileOptions {
+        sourceCompatibility = 1.8
+        targetCompatibility = 1.8
+    }
+    lintOptions {
+        abortOnError false
```

Comment:
What errors are we disabling here?

Reply:
When building using the terminal, it will fail because of formatting errors. This is what's disabled here

Reply:
What formatting errors? Why can't we fix those errors instead?

Reply:
You can see the error here: https://github.com/RomainL972/passport-reader/runs/964036499
I don't really know how to fix this

Reply:
Something like this: https://stackoverflow.com/a/50169130/2548565

Reply:
Also, maybe we can get rid of `apache.commons`? I'm sure there is a way to accomplish same tasks without dragging extra dependencies.

Reply:
I've added a lint.xml file and it now works

Reply:
The apache.commons dependency is used to capitalize names like "LAST NAME" => "Last Name"
I guess I could write a function that does the same but it seems like some unnecessary work.